### PR TITLE
Remove unnecessary using statements

### PR DESCRIPTION
**Pull request by programmeruser2**

### CODE REVIEW

**Review by darkdarcool — APPROVED** [review]
LGTM. Let's wait until @whippingdot, can get back on to merge tho.

File: C++/0. Setup.md
Comment (whippingdot) [inline C++/0. Setup.md:14]: umm isn't this fine? I didn't write `using namespace std;`. This only imports the specific file. It is easier than writing std:: before everything...

**Comment by whippingdot** [comment]
hey @programmeruser2 i dont think this is needed. Tell me ur aspect of why you would rather use `std::` before everything instead of just importing that one object using `using std::`.

**Comment by programmeruser2** [comment]
Is it really that hard to type `std::`? And `using` directives can still introduce name clashes even if you only use `using` for certain namespace members.

**Comment by whippingdot** [comment]
> Is it really that hard to type `std::`? And `using` directives can still introduce name clashes even if you only use `using` for certain namespace members.

koi koi. Also yes it is lul. Ima still use the using thingy but if I run into a problem i will just use std::

**Comment by darkdarcool** [comment]
`std::` is important and is not hard to type for me, and I kinda like typing it. std::, std::

**Comment by whippingdot** [comment]
hey @programmeruser2 if u r gonna help frequently you could join the repo...

**Pull request merged**
